.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-python: 3.6
+python: 3.9
```

Comment:
Why keep both Travis CI and GitHub Actions?

Reply:
Github Actions are not yet working properly.

Reply:
https://github.com/scrapinghub/shub/pull/407#discussion_r716021976